integration-libs/opps/root/coupon-codes/opps-coupon-codes.module.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { LOCATION_INITIALIZED_MULTI } from '@spartacus/core';
+import { oppsCouponCodesInterceptors } from './http-interceptors';
+import { OppsCouponCodesService } from './opps-coupon-codes.service';
+
+export function saveCouponCodesFactory(
+  service: OppsCouponCodesService
+): () => void {
+  return () => {
```

Comment:
nitpick: for better type safety, let's use `inject()` function of Angular in the top of the factory funciton body, instead of passing `deps[]` array in line 27

```suggestion
export function saveCouponCodesFactory(): () => void {
  const service = inject(OppsCouponCodesService);
  return () => {
```

Reply:
done